test/utils/exceptions_test.dart

```diff
@@ -65,6 +70,28 @@ void main() {
           clearTrailingDot ? 'Max daily limit spent' : 'Max daily limit spent.',
         );
       });
+
+      test('high fees (clearTrailingDot=$clearTrailingDot)', () {
+        expect(
+          extractExceptionMessage(
+            PlatformException(
+              code: "Method Error",
+              message: "Error Domain=go Code=1 \"rpc error: code = Unknown desc = fees are to high for the "
+                  "given amount transaction output amount is negative\" UserInfo={NSLocalizedDescription=rpc "
+                  "error: code = Unknown desc = fees are to high for the given amount transaction output "
+                  "amount is negative}",
+              details: "Error Domain=go Code=1 \"rpc error: code = Unknown desc = fees are to high for the "
+                  "given amount transaction output amount is negative\" UserInfo={NSLocalizedDescription=rpc "
+                  "error: code = Unknown desc = fees are to high for the given amount transaction output "
+                  "amount is negative}",
+            ),
+            clearTrailingDot: clearTrailingDot,
+          ),
+          // There is a typo in the error message, "to high" instead of "too high" as we are replacing the
+          // whole message with a localized one it is not a problem for our end user
+          "fees are to high for the given amount transaction output amount is negative",
```

Comment:
Hey @ademar111190,
You can replace this with the correct version. @yaslama has fixed the typo on our end earlier today. See https://github.com/breez/lnd/commit/99fa19e3e643bd95e3ff8f8f8d7f94ec15e87740 which should be live as of today.

There are two things I don't like about our localized message,
- It doesn't have "transaction output amount is negative" information, which is a dynamic value as you can see from the implementation,
- It's `smallcase`, if we'll be using a localized message `Sentence case` would be better suited.

Reply:
I've also added the "too high", so if the only change were that, it would not be a problem.

Reply:
Removed the to high